.github/workflows/CI.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
Note that this runs CI testing against all branches to (so runs testing when you push to branches.) I like this feature, but if you don't we can maintain he current behavior of just testing merges to master and PRs by doing:

```yml
on:
  push:
    branches: [ master ]
  pull_request:
```